source/special.tex

```diff
@@ -3372,46 +3372,46 @@
 of the defaulted three-way comparison operator function
 with parameters \tcode{x} and \tcode{y} of the same type
 is determined by comparing corresponding elements
-\tcode{x}$_i$ and \tcode{y}$_i$
+$\tcode{x}_i$ and $\tcode{y}_i$
 in the expanded lists of subobjects for \tcode{x} and \tcode{y}
 until the first index $i$
-where \tcode{x}$_i$ \tcode{<=>} \tcode{y}$_i$
-yields a result value \tcode{v}$_i$ where \tcode{v}$_i$\tcode{ != 0},
+where $\tcode{x}_i$ \tcode{<=>} $\tcode{y}_i$
+yields a result value $\tcode{v}_i$ where $\tcode{v}_i \tcode{!= } 0$,
 contextually converted to \tcode{bool}, yields \tcode{true};
-\tcode{V} is \tcode{v}$_i$ converted to \tcode{R}.
+\tcode{V} is $\tcode{v}_i$ converted to \tcode{R}.
 If no such index exists, \tcode{V} is
 \tcode{std::strong_ordering::equal} converted to \tcode{R}.
 
 \pnum
 The \defn{common comparison type} \tcode{U}
 of a possibly-empty list of $n$ types
-\tcode{T}$_0$, \tcode{T}$_1$, $\cdots$, \tcode{T}$_{n-1}$
+$\tcode{T}_0$, $\tcode{T}_1$, $\dotsc$, $\tcode{T}_{n-1}$
```

Comment:
I think I'd prefer a non-math \ldots here, since we don't have math commas.

Reply:
We're doing this style elsewhere, too: "$ grep dotsc *.tex"

Reply:
Hm, that's unfortunate, but fine.